ui/hooks/useUserPreferencedCurrency.js

```diff
@@ -41,14 +42,22 @@ import { ETH_DEFAULT_DECIMALS } from '../constants';
  * @returns {UserPreferredCurrency}
  */
 export function useUserPreferencedCurrency(type, opts = {}) {
-  const nativeCurrency = useSelector(getMultichainNativeCurrency);
+  const selectedAccount = useSelector(getSelectedInternalAccount);
+  const account = opts.account || selectedAccount;
```

Comment:
`opts.account` should never be `null` here, so let's use `??` instead:

```suggestion
  const account = opts.account ?? selectedAccount;
```